test/api_integration/apis/saved_objects/find.js

```diff
@@ -88,7 +88,7 @@ export default function ({ getService }) {
       describe('unknown search field', () => {
         it('should return 200 with empty response', async () => (
           await supertest
-            .get('/api/saved_objects/_find?type=wigwags&search_fields=a')
+            .get('/api/saved_objects/_find?type=url&search_fields=a')
```

Comment:
Would you mind elaborating on why this change was necessary?

Reply:
This could probably be reverted but, what happened was that the validation rules were run prior to the checking of the search field a bad request error was thrown before any checking of the search fields. But I am a little confused on the objective of the test for the search field?